.github/workflows/test.yml

```diff
@@ -0,0 +1,46 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macOS-latest]
+        python-version: [3.5, 3.6, 3.7, 3.8, 3.9]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Installation
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+        python setup.py install
+    - name: Test requirements Installation
+      run: |
+        python otherfiles/requirements-splitter.py
+        pip install --upgrade --upgrade-strategy=only-if-needed -r test-requirements.txt
+    - name: First test
+      run: |
+        python -m pytest test --cov=penney --cov-report=term
+    - name: Version check
+      run: |
+        python version_check.py
+      if: matrix.python-version == 3.7
+    - name: Other tests
+      run: |
+          python -m vulture --min-confidence 65 --exclude=__init__.py --sort-by-size penney/ setup.py version_check.py
+          python -m bandit -r penney -s B311
+          python -m pydocstyle --match-dir=penney
+      if: matrix.python-version == 3.7
+    - name: Codecov
```

Comment:
Please add a condition to this section to call `codecov` only in (`ubuntu` && `python 3.7`)